lib/posix/posix.nim

```diff
@@ -1124,7 +1124,7 @@ var
     importc: "_CS_POSIX_V6_LPBIG_OFFBIG_LIBS", header: "<unistd.h>".}: cint
   CS_POSIX_V6_WIDTH_RESTRICTED_ENVS* {.
     importc: "_CS_POSIX_V6_WIDTH_RESTRICTED_ENVS", header: "<unistd.h>".}: cint
-  F_LOCK* {.importc: "F_LOCK", header: "<unistd.h>".}: cint
+  f_lock* {.importc: "F_LOCK", header: "<unistd.h>".}: cint
   F_TEST* {.importc: "F_TEST", header: "<unistd.h>".}: cint
```

Comment:
@dom96 I'm ready to go on changing all the procs but I wanted to get an official word since there are so many. If this is the green light, I'll take care of it.

 Also, there are many, many variables that are all caps. I can do those too. There'll be a ton of deprecation. Don't know if the idea is to fix all names that don't meet the guidelines or not.